doc/tommath.src

```diff
@@ -5986,8 +5949,6 @@ are congruent to one modulo four, otherwise it evaluates to negative one.
 By step nine if $a'$ does not equal one a recursion is required.  Step 9.1 computes $p' \equiv p \mbox{ (mod }a'\mbox{)}$ and will recurse to compute
 $\left ( {p' \over a'} \right )$ which is multiplied against the current Jacobi product.
 
-EXAM,bn_mp_jacobi.c
```

Comment:
I really don't like how the book is torn apart... we should probably remove it entirely since it won't be updated... another option would be to fix those parts, but I'm not sure if it's worth the effort.

Reply:
Yes, I would probably remove it and freeze it. It is a nice thing but it seems tommath diverged quite a bit already from what it was in the beginning. It is still useful as a reference.

Reply:
The thing is - the functions I was touching until now were quite trivial. But this more complicated number theoretic stuff @czurnieden is doing really needs docs.

Reply:
@sjaeckel I removed the doc changes for now.

Reply:
> The thing is - the functions I was touching until now were quite trivial

this applies to my changes as well...

> But this more complicated number theoretic stuff @czurnieden is doing really needs docs.

@czurnieden already updates the regular documentation... I don't know if this book should be updated as well... probably @czurnieden can answer this?

Reply:
I don't get what you mean by regular docs vs book. I think the book is the docs - at least this is what I touched for example in #279 too, when @czurnieden asked for docs.

Reply:
> You mean bn.tex vs tommath.src?

yep

Reply:
Let's make this compromise:

* We don't touch the book tommath.src/tex except for making sure that it compiles (Right now it does not)
* We update bn.tex

Reply:
I would say

* We remove the book, since a compiling, but not up-to-date book doesn't help
* We update bn.tex

I'd go then and make a build of the latest up-to-date version of the book and add that one on the libtommath page

Reply:
That's also ok I guess.

Reply:
> That's also ok I guess.

deal!

@czurnieden should be fine with this as well, as he was also updating bn.tex all the time